b2/sync/policy.py

```diff
@@ -152,11 +152,10 @@ class DownPolicy(AbstractFileSyncPolicy):
 
     def _make_transfer_action(self):
         return B2DownloadAction(
-            self._source_file.name,
-            self._source_folder.make_full_path(self._source_file.name),
+            self._source_file.name, self._source_folder.make_full_path(self._source_file.name),
             self._source_file.latest_version().id_,
-            self._dest_folder.make_full_path(self._source_file.name),
-            self._get_source_mod_time(), self._source_file.latest_version().size
+            self._dest_folder.make_full_path(self._source_file.name), self._get_source_mod_time(),
+            self._source_file.latest_version().size
```

Comment:
this gets poorly formatted due to missing , at the end of last value, please correct it